fe1-web/components/eventList/LaoProperties.tsx

```diff
@@ -10,12 +10,38 @@ import { useSelector } from 'react-redux';
 import ParagraphBlock from 'components/ParagraphBlock';
 import { Lao } from 'model/objects';
 import { makeCurrentLao } from 'store/reducers';
+import EdiText from 'react-editext';
 
 function renderProperties(lao: Lao) {
+  const style = {
+    fontFamily: 'Helvetica Bold',
```

Comment:
<img width="295" alt="Screenshot 2021-03-10 at 01 34 55" src="https://user-images.githubusercontent.com/44499915/110557514-03b54200-8141-11eb-8442-a1797d07428a.png">


Any reason why you decided to change the font for those two lines only? ^^'

Reply:
Hi no real reason actually. I wanted to change the one you posted here in this picture because it was Times New Roman instead of Helvetica. Then I changed it to bold, but I'll put it back to normal Helvetica!